.gitignore

```diff
@@ -18,6 +18,7 @@
 # Ignore the default SQLite database.
 /db/*.sqlite3
 /db/*.sqlite3-journal
+/db/structure.sql
```

Comment:
This is not acceptable, the `structure.sql` is an integral part of the application and should not be ignored in commits.